include/benchmark/benchmark.h

```diff
@@ -153,14 +154,18 @@ void Initialize(int* argc, const char** argv);
 
 // Otherwise, run all benchmarks specified by the --benchmark_filter flag,
 // and exit after running the benchmarks.
-void RunSpecifiedBenchmarks(const BenchmarkReporter* reporter = nullptr);
+void RunSpecifiedBenchmarks(const BenchmarkReporter* reporter = NULL);
```

Comment:
I tried to leave it alone inside the library source. However I would like to keep the headers compiling in C++03 so I can benchmark stuff in C++03.